Paymetheus.Tests.Decred/Sha256Tests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Paymetheus.Decred.Wallet;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Paymetheus.Tests.Decred
+{
+    public static class Sha256Tests
```

Comment:
Since I commented above that I would prefer the DoubleSha256 method to be part of the WalletSeed class, this file and class should be renamed to WalletSeedTests.

You can test a private (or protected) method in another assembly by declaring this assembly as a friend.  Use the [InternalsVisibleToAttribute](https://msdn.microsoft.com/en-us/library/system.runtime.compilerservices.internalsvisibletoattribute%28v=vs.110%29.aspx?f=255&MSPPError=-2147217396).  For example:

```c#
[InternalsVisibleTo("Paymetheus.Tests.Decred")]
private static byte[] DoubleSha256(byte[] value)
{
    // ...
}
```